### PR TITLE
feat(1922): Add openPr method. BREAKING CHANGE: pull in new scm-base

### PR DESCRIPTION
**Pull request by klu909**
## Context

Sd onboarding feature requires an api to open pull request

## Objective

Add a method openPr for opening pull request to support sd onboarding feature

## References

https://github.com/screwdriver-cd/screwdriver/issues/1922

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
Need to pull in new major version of scm-base: https://github.com/screwdriver-cd/scm-router/blob/master/package.json#L46
and bump major here

**Comment by tkyi** [comment]
Can you remove package-lock file?

File: test/index.test.js [hunk 1]
```diff
@@ -980,4 +980,36 @@ describe('index test', () => {
                 });
         });
     });
+
+    describe('_openPr', () => {
```
Comment (tkyi) [inline test/index.test.js:985]: test is failing because you need to mock this function

Comment (klu909) [reply test/index.test.js:985]: thanks I fixed it

**Review by tkyi — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**